### ISSUE TITLE
Test (/Fix?) Color Contrast in Figure Captions

### ISSUE THREAD

**Issue by openstaxalina** [openstax/cnx-recipes#4296]
# Brief Description of Change and Screenshots
What kind of change is this? [ ] recipe [x] styles [ ] i don't know

![Screen Shot 2021-12-07 at 4 47 54 PM](https://user-images.githubusercontent.com/7863733/145117685-46e08134-2fe7-41a5-8ab1-f5f28460eba0.png)

I captured the hex value for the figure caption as #a4a3a3. According to WCAG this fails against a white background:

![Screen Shot 2021-12-07 at 4 48 04 PM](https://user-images.githubusercontent.com/7863733/145117688-131c6478-06e0-4f98-b173-6e22747c9e74.png)

Please first check with a dev (?) what the hex value is, and retest using the link under "Acceptance Criteria". If it fails, we need to determine a new hex value that passes.

# Acceptance Criteria
- [ ] figure captions pass WCAG 2.1 AA contrast criteria for normal text. can be tested here: https://webaim.org/resources/contrastchecker/

# Content Location(s)
collection ID:
github openstax/osbooks- link: github.com/openstax/os-books/intro-finance
chapter/section: 14.2
surrounding text: "further away from the line"

**Comment by lwtchu** [comment]
@openstaxalina I got `#747373` when I checked the grey - which does pass.  How are you pulling the hex value?

**Comment by openstaxalina** [comment]
@lwtchu thank you for looking into this. I was using an eye dropper tool, which I know can be imprecise, hence my suggestion to first check the actual HEX value. Is #747373 the true value set in the styles code?

**Comment by lwtchu** [comment]
@openstaxalina yep! well technically it's set to Grey2: #757474, - but adobe renders to the hex value 747373 - you can edit the PDF in adobe, click into the text and the color should pop up in the edit text format tool bar on the right and you c an check the hex value there

**Comment by openstaxalina** [comment]
gotcha - this can be closed then - closing!
